pallets/pallet-bonded-coins/src/curves/mod.rs

```diff
@@ -23,9 +23,9 @@
 /// - Polynomial
 /// - SquareRoot
 /// - LMSR (Logarithmic Market Scoring Rule)
-pub(crate) mod lmsr;
-pub(crate) mod polynomial;
-pub(crate) mod square_root;
+pub mod lmsr;
```

Comment:
All the types in there already have the `LSMR` prefix, so ideally they would need to be re-exported directly instead of being namespaced by the module name. But this is a very unimportant thing to do, so just go ahead for now.